src/footer/Footer.js

```diff
@@ -12,32 +14,70 @@ const FooterContainer = styled.footer`
 const FooterContent = styled.div`
   align-items: center;
   display: flex;
-  justify-content: space-between;
-  margin: 0 auto;
-  max-width: ${CONTAINER_WIDTH}px;
+  flex-direction: column;
   min-height: ${(props) => props.theme.spacing[74]};
+
+  ${breakpoint("md")`
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 0 auto;
+    max-width: ${CONTAINER_WIDTH}px;
+  `}
 `;
 
 const FooterCredits = styled.div`
-  min-height: ${(props) => props.theme.spacing[24]};
-  width: 40%;
+  padding: ${(props) => props.theme.spacing[8]};
+
+  ${breakpoint("md")`
+    min-height: ${(props) => props.theme.spacing[24]};
+    width: 40%;
+  `}
+
+  /* 
+    There is a bug in styled-components-breakpoint libary which doesn't allow multiple
+    interpreted values to be used within the same 'breakpoint' block:
+    https://github.com/jameslnewell/styled-components-breakpoint/issues/26
+
+    As a workaround, it is possible to achieve the desired result by using a separate
+    breakpoint block for each interpreted property.
+  */
```

Comment:
This is 💩 .  There is an [alternative library](https://github.com/mg901/styled-breakpoints), but I didn't like [its API](https://github.com/mg901/styled-breakpoints#api).

So there are really two discussion points:

1.  Do we like this overall approach to handling mobile styles?
2.  Separately, is this duplicate code workaround reasonable or should we find another library (or fork and fix the current library)